src/lib/serviceRoutines/postQueryContext.cpp

```diff
@@ -297,13 +297,49 @@ std::string postQueryContext
   std::string                 answer;
   QueryContextRequestVector   requestV;
   QueryContextResponseVector  responseV;
+  long long                   count;
+  long long*                  countP = NULL;
+
+
+  //
+  // 00. Count or not count? That is the question ...
+  //
+  // For API version 1, if the URI parameter 'details' is set to 'on', then the total of local
+  // entities is returned in the errorCode of the payload.
+  //
+  // In API version 2, this has changed completely. Here, the total count of local entities is returned
+  // if the URI parameter 'count' is set to 'on', and it is returned in the HTTP header X-Total-Count.
```

Comment:
Actually, is "true" and not "on" in this case (see https://github.com/telefonicaid/fiware-orion/issues/947)